environment

```diff
@@ -90,3 +91,6 @@ set +a
 if [[ -f "${DSS_HOME}/environment.local" ]]; then
     source "${DSS_HOME}/environment.local"
 fi
+
+scripts/get_gcp_credentials.py || echo "Unable to recover gcp credentials from Secrets Manager. Your configuratoin is incorrect/incomplete."
+export ADMIN_USER_EMAILS=`jq ".client_email" ${GOOGLE_APPLICATION_CREDENTIALS} | sed 's/"//g'` || ADMIN_USER_EMAILS=""
```

Comment:
`jq -r` omits the quotes, I think, and that may allow you to skip the `sed` stage. This is not only an optimization: email addresses can contain double quotes, and they'd be escaped in the JSON string. The `sed` would replace them  indiscriminately.

https://tools.ietf.org/html/rfc5322#page-17

Search for definition of `quoted-string`

I'm aware that this is not your code originally but the fix is straight-forward so I thought I'd mention it.

Reply:
Dropped the `sed`